### ISSUE TITLE
Internationalisation

### ISSUE THREAD

**Issue by CandiceJoy** [studiokotatsu/moose#7]
Language translation

**Comment by CandiceJoy** [comment]
Language translation, unit conversions, maybe even let people pick default units and such and the bot will display those when doing DM stuff, time conversions, etc.

Might even get fancy and have it auto-parse messages for things that aren't english, for units, for times, etc...and do a little fancy auto-conversion.  Maybe even let it pick up timezones and units and things based on user preferences.

Example (angled brackets are comments, not i/o)
User: Es ist 21:00 und 21C in Deutschland.
Bot: It is (8:00pm UTC) and (69.8F) in (Germany). <It used the location to translate the timezone...otherwise would just say 9:00pm without timezone>

Maybe allow people to setup particularly technical or jargony terms that auto-explain too.

Aussie: Hop in me ute!
Bot: Ute - utility truck

Brit: I'm fucking pissed mate!
Bot: Pissed (American) - Angry   /   Pissed (British) - Drunk

Basically, a sort of moderate cultural translator to increase the ability for people across the world to understand each other.
